navit/navit.c

```diff
@@ -3504,6 +3504,27 @@ void navit_layout_switch(struct navit *n) {
                 && valid_attr.u.num==attr_position_valid_invalid) {
             return; //No valid fix yet
         }
+
+        int *streetflags = tracking_get_current_flags(n->tracking);
```

Comment:
Suggest to move variable declaration to top of function. We still support platforms with old C compilers not supporting this.